src/main/java/edu/hawaii/its/api/service/OotbGroupingPropertiesService.java

```diff
@@ -55,53 +54,65 @@
     private String GROUPING_OOTBS;
 
     @Value("${groupings.api.grouping_admins}")
     private String GROUPING_ADMINS;
 
-    @Autowired
     @Qualifier("HasMembersResultsOOTBBean")
-    private HasMembersResults hasMembersResults;
+    private final HasMembersResults hasMembersResults;
 
-    @Autowired
     @Qualifier("FindGroupsResultsOOTBBean")
-    private FindGroupsResults findGroupsResults;
+    private final FindGroupsResults findGroupsResults;
 
-    @Autowired
     @Qualifier("GetSubjectsResultsOOTBBean")
-    private SubjectsResults subjectsResults;
+    private final SubjectsResults subjectsResults;
 
-    @Autowired
     @Qualifier("GroupSaveResultsOOTBBean")
-    private GroupSaveResults groupSaveResults;
+    private final GroupSaveResults groupSaveResults;
 
-    @Autowired
     @Qualifier("AssignAttributesOOTBBean")
-    private AssignAttributesResults assignAttributesResults;
+    private final AssignAttributesResults assignAttributesResults;
 
-    @Autowired
     @Qualifier("GetMembersResultsOOTBBean")
-    private GetMembersResults getMembersResults;
+    private final GetMembersResults getMembersResults;
 
-    @Autowired
     @Qualifier("AddMemberResultsOOTBBean")
-    private AddMembersResults addMembersResults;
+    private final AddMembersResults addMembersResults;
 
-    @Autowired
     @Qualifier("RemoveMembersResultsOOTBBean")
-    private RemoveMembersResults removeMembersResults;
+    private final RemoveMembersResults removeMembersResults;
 
-    @Autowired
     @Qualifier("AttributeAssignmentResultsOOTBBean")
-    private GroupAttributeResults groupAttributeResults;
+    private final GroupAttributeResults groupAttributeResults;
 
-    @Autowired
     @Qualifier("GetGroupsResultsOOTBBean")
-    private GetGroupsResults getGroupsResults;
+    private final GetGroupsResults getGroupsResults;
+
+    public OotbGroupingPropertiesService(HasMembersResults hasMembersResults,
+            FindGroupsResults findGroupsResults,
+            SubjectsResults subjectsResults,
+            GroupSaveResults groupSaveResults,
+            AssignAttributesResults assignAttributesResults,
+            GetMembersResults getMembersResults,
+            AddMembersResults addMembersResults,
+            RemoveMembersResults removeMembersResults,
+            GroupAttributeResults groupAttributeResults,
+            GetGroupsResults getGroupsResults) {
+        this.hasMembersResults = hasMembersResults;
+        this.findGroupsResults = findGroupsResults;
+        this.subjectsResults = subjectsResults;
+        this.groupSaveResults = groupSaveResults;
+        this.assignAttributesResults = assignAttributesResults;
+        this.getMembersResults = getMembersResults;
+        this.addMembersResults = addMembersResults;
+        this.removeMembersResults = removeMembersResults;
+        this.groupAttributeResults = groupAttributeResults;
+        this.getGroupsResults = getGroupsResults;
+    }
```

Comment:
I'm curious if the @Qualifier annotation is enough and we don't need to inject results at all.